pkg/cert/parsecerts.go

```diff
@@ -0,0 +1,44 @@
+package cert
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+)
+
+var ErrNoValidPEMCertificatesFound = errors.New("no valid PEM certificates found")
+
+// ParseCerts parses a PEM-encoded certificate chain formatted as concatenated strings
+// and returns a slice of x509.Certificate. In case of unparsable certificates, the
+// function returns an empty slice.
```

Comment:
(niptick) should this also say that an error is returned? I.e. `In case of unparsable certificates, the
// function returns an empty slice and an error`?